packages/cozy-procedures/src/components/Documents.jsx

```diff
@@ -6,20 +6,29 @@ import { Title, translate, Label, Button } from 'cozy-ui/transpiled/react/'
 import EmptyDocumentHolder from './documents/EmptyDocumentHolder'
 import LoadingDocumentHolder from './documents/LoadingDocumentHolder'
 import DocumentHolder from './documents/DocumentHolder'
+import creditApplicationTemplate from '../templates/creditApplicationTemplate'
 
 class Documents extends React.Component {
   render() {
     const { t, router } = this.props
+    const { documents } = creditApplicationTemplate
```

Comment:
Should we rename it to documentsTemplate ? 

I would prefer to have `documents` to refer to the "selected" documents / files ?

Reply:
Yep.